ivy/functional/backends/tensorflow/layers.py

```diff
@@ -491,3 +491,22 @@ def conv_general_transpose(
     if data_format == "channel_first":
         res = tf.transpose(res, (0, dims + 1, *range(1, dims + 1)))
     return res
+
+
+@to_native_arrays_and_back
```

Comment:
this decorator should be added to the `mixed_backend_wrappers` attribute of the `ivy` function like done for [layer_norm](https://github.com/unifyai/ivy/blob/dca2ac408963d436c61cc1ab3cc6cb641959611e/ivy/functional/ivy/norms.py#L142)